modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Containers.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2022-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.kinesis
+
+import org.slf4j.LoggerFactory
+
+import org.testcontainers.containers.{BindMode, GenericContainer => JGenericContainer, Network}
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.containers.output.Slf4jLogConsumer
+
+import com.dimafeng.testcontainers.{FixedHostPortGenericContainer, GenericContainer}
+
+object Containers {
+
+  val (localstack, localtackNetwork) = {
+    val container = FixedHostPortGenericContainer(
+      imageName = "localstack/localstack-light:1.0.4",
+      exposedPorts = Seq(4566),
+      fileSystemBind = Seq(
+        GenericContainer.FileSystemBind(
+          "modules/kinesis/src/it/resources/localstack",
+          "/docker-entrypoint-initaws.d",
+          BindMode.READ_ONLY
+        )
+      ),
+      waitStrategy = Wait.forLogMessage(".*AWS kinesis.CreateStream.*", 6),
+      exposedHostPort = 4566,
+      exposedContainerPort = 4566
```

Comment:
This is a leftover from previous implementation without the custom network. I wonder if all the containers of a same network can access each other's ports, I'll try!

Reply:
Now I remember why the `FixedHostPortGenericContainer`. We don't need it for Enrich and localstack to communicate. But we do need it for [Kinesis producer and consumer](https://github.com/snowplow/enrich/blob/feature/integration_test_kinesis/modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/ConfigCount.scala#L29-L31) that sends and reads data for the integration test `count`.

Reply:
Yes, but with non-fixed version you can still call `container.getMappedPort(4566)` to get randomly assigned port and use it to configure producer/consumer dynamically in tests instead of hardcoded value?

Reply:
Ah I now get what you said in your first comment! Good idea, it should work!